profile/README.md

```diff
@@ -0,0 +1,17 @@
+<p align="center" >
+<img src="https://docs.dash.org/en/stable/_images/dash_logo.png" alt="Dash Wallet logo" title="Dash Wallet" width="300">
+</p>
+
+WELCOME TO DASHPAY!!
+
+The DashPay organization houses all GitHub activity related to Dash.
+
+Previously, all development related to Dash Platform happened under the dashevo GitHub. However, we have since archived the repositories in Dashevo and moved everything to the DashPay GitHub.
```

Comment:
```suggestion
Previously, all development related to Dash Platform happened under the [dashevo](https://github.com/dashevo/) GitHub organization. However, we have since archived the repositories in dashevo and moved everything to the current dashpay GitHub organization.
```